src/ir/ir-builder.h

```diff
@@ -31,29 +31,56 @@ class IRBuilderWrapper: public Nan::ObjectWrap, public FromValueMixin<IRBuilderW
     template<ConvertOperationFn method>
     static NAN_METHOD(ConvertOperation);
 
+    static NAN_METHOD(CreateAdd);
```

Comment:
@MichaReiser 
![image](https://user-images.githubusercontent.com/572096/47761776-24616780-dcfd-11e8-973b-fb2039afa89f.png)

https://github.com/nodejs/nan/blob/master/doc/methods.md#method-declaration

Should we drop it ( NAN v1 method declarations.)?

Reply:
Thanks for the input. Wasn't aware that the Macro isn't longer the suggested way. Let us look at this in a separate issue/PR.